<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>377</width>
      <height>316</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_27">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout_24">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout_9">
         <item row="0" column="0">
          <layout class="QFormLayout" name="formLayout_19">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>HW 1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QFormLayout" name="formLayout_17">
             <item row="0" column="0">
              <widget class="QSlider" name="horizontalSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QFormLayout" name="formLayout_18">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>HW 2</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QFormLayout" name="formLayout_16">
             <item row="0" column="0">
              <widget class="QSlider" name="horizontalSlider_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox_2"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QFormLayout" name="formLayout_15">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>HW 3</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QFormLayout" name="formLayout_14">
             <item row="0" column="0">
              <widget class="QSlider" name="horizontalSlider_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox_3"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QFormLayout" name="formLayout_13">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>HW 4</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QFormLayout" name="formLayout_12">
             <item row="0" column="0">
              <widget class="QSlider" name="horizontalSlider_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox_4"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <layout class="QFormLayout" name="formLayout_11">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>HW 5</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QFormLayout" name="formLayout_10">
             <item row="0" column="0">
              <widget class="QSlider" name="horizontalSlider_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox_5"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <layout class="QFormLayout" name="formLayout_8">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>HW 6</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QFormLayout" name="formLayout_7">
             <item row="0" column="0">
              <widget class="QSlider" name="horizontalSlider_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox_6"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <layout class="QFormLayout" name="formLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>HW 7</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QFormLayout" name="formLayout_5">
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox_7"/>
             </item>
             <item row="0" column="0">
              <widget class="QSlider" name="horizontalSlider_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="7" column="0">
          <layout class="QFormLayout" name="formLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>HW 8</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QSlider" name="horizontalSlider_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox_8"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QFormLayout" name="formLayout_23">
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>78</width>
             <height>208</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <layout class="QFormLayout" name="formLayout_22">
           <item row="0" column="0">
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Midterm One</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QSlider" name="horizontalSlider_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QSpinBox" name="spinBox_9"/>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QFormLayout" name="formLayout_20">
             <item row="0" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Midterm Two</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QSlider" name="horizontalSlider_11">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QSpinBox" name="spinBox_10"/>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QFormLayout" name="formLayout_21">
             <item row="0" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Final Exam</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QSlider" name="horizontalSlider_10">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="QSpinBox" name="spinBox_11"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QFormLayout" name="formLayout_26">
       <item row="0" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>358</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <layout class="QFormLayout" name="formLayout_25">
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>FINAL GRADE</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_12"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGrade_Calculator">
    <property name="title">
     <string>Grade Calculator</string>
    </property>
   </widget>
   <addaction name="menuGrade_Calculator"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_5</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_6</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_7</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_8</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_9</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_9</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_11</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_10</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_10</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_11</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_9</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_10</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_11</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_11</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_10</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
